### DISCUSSION TITLE
【作业提交】zkiap session 2

### DISCUSSION THREAD

**Discussion by Demian101** [zkp-co-learning/zkp-co-learn#30] [Show and tell]
把你的作业 repo 提交到评论区吧~

1、office hour 的时候可以直接打开某条作业点评
2、便于从 participates 区域统计大家的作业完成情况

**Comment by Howard-Hu** [comment]
【作业提交】https://github.com/Chris-Howard/zkiapExercise

**Comment by Insomnia501** [comment]
https://github.com/Insomnia501/circom-zkp/blob/main/exercise2.circom

**Comment by xiegudong45** [comment]
https://github.com/xiegudong45/MIT-IAP/tree/main/session2

**Comment by izayl** [comment]
https://github.com/izayl/mit-zkiap/blob/main/exercises/2.md

**Comment by abbylow** [comment]
https://github.com/abbylow/zkiap/tree/main/session-2-exercise
